src/credential/credential.service.ts

```diff
@@ -54,9 +57,9 @@ export class CredentialService {
 
   public async revokeCredential(credential_exchange_id: string, dto: RevokeCredentialDto): Promise<Credential> {
     return this.credentialRepo.manager.transaction(async (entityManager: EntityManager) => {
-      let cred: Credential = await entityManager.findOne(Credential, {credential_exchange_id});
+      const cred: Credential = await entityManager.findOne(Credential, {credential_exchange_id});
       if (!cred) {
-        throw('Credential not found in database with credential exchange id: ' + credential_exchange_id)
+        throw(new Error('Credential not found in database with credential exchange id: ' + credential_exchange_id));
```

Comment:
I'm wondering if we should make this a ProtocolException - the advantage being that they will then be formatted in such a way that the caller of this microservice could handle it in different ways if needed. Although to take full advantage you'd need to add the ProtocolExceptionFilter as a global filter like so:
https://github.com/kiva/aries-guardianship-agency/blob/master/src/app/app.service.ts#L25

And then maybe while you're there you could add Datadog logging like so:
https://github.com/kiva/aries-guardianship-agency/blob/master/src/app/app.service.ts#L21

Reply:
Yeah I was just fixing some quick linting errors here. We can make this more conformant to our observability standards in a follow up. I need to add tracing too.

Reply:
I'll write up a ticket about it